### PR TITLE
Update latest releases link in README.md

### LINKED ISSUE #144: Dummy issue: For testing the closing of issues from PR.
- Dummy issue.

### PR DESCRIPTION
**Pull request by MithunKinarullathil**
- Update README
- Unfortunately, I am unable to find a dynamic link to the latest Debian file. @AlexFernandes-MOVAI Any black magic under your sleeves? 
- Fixes #144 
- Requires #145 

- [x] Make sure you are opening from a **topic/feature/bugfix branch**
- [x] Ensure that the PR title represents the desired changes
- [x] Ensure that the PR description detail the desired changes
- [ ] ~~Link to relevant issues in GitHub or Jira~~
- [ ] ~~Link to relevant pull requests, esp. upstream and downstream changes~~
- [ ] ~~Ensure you have provided tests - that demonstrates feature works or fixes the issue~~

[^note]:
    Put an `x` into the [ ] to show you have filled the information.
    The template comes from https://github.com/MOV-AI/.github/blob/master/.github/pull_request_template.md
    You can override it by creating .github/pull_request_template.md  in your own repository

### CODE REVIEW

**Comment by AlexFernandes-MOVAI** [comment]
`curl -s https://api.github.com/repos/MOV-AI/movai-flow/releases/latest | grep "browser_download_url.*deb" | cut -d : -f 2,3 | tr -d \"`

**Review by AlexFernandes-MOVAI — COMMENTED** [review]
You can also add a badge on top of the page:
[github
release](https://github.com/MOV-AI/movai-flow/releases)
`[github
release](https://github.com/MOV-AI/movai-flow/releases)`

File: README.md [hunk 1]
```diff
@@ -74,65 +94,57 @@ sudo apt-get update && sudo apt-get install -y nvidia-container-toolkit
...
+# Installation
+## Download and Install
+##### Option A)
+- Download the debian file from the **Assets** section of the [latest releases page](https://github.com/MOV-AI/movai-flow/releases/latest)
+- Click on the downloaded file in your browser or double click on the `movai-flow*.deb` file in your Downloads directory using a file explorer
+- Enter your user password if there is a promt.
+- Click the Install option. The Ubuntu Software Package Manager starts installing the downloaded app.
 
+##### Option B)
+Open a terminal, down the latest debian file and install it. Enter your user password if there is a promt.
```
Comment (AlexFernandes-MOVAI) [inline README.md]: download instead of  `down`

File: README.md [hunk 2]
```diff
@@ -74,65 +94,57 @@ sudo apt-get update && sudo apt-get install -y nvidia-container-toolkit
...
-MOV.AI Flow&trade; initiates a set of services running as a `docker-compose` cluster, namely:
+MOV.AI Flow™ initiates a set of services running as a *docker-compose* cluster, namely:
 
  - redis-master: master DB of the cluster
  - redis-local: local DB of the cluster
  - backend: web service application
  - ros-master: ROS core service
  - spawner: Flow orchestrator of ROS nodes
- - haproxy: web proxy
  - simulator: Ignition Fortress container with graphical capabilities enabled
```
Comment (AlexFernandes-MOVAI) [inline README.md:143]: why removing haproxy

File: README.md [hunk 3]
```diff
@@ -142,8 +154,48 @@ MOV.AI Flow&trade; initiates a set of services running as a `docker-compose` clu
...
+    limitations under the License.
+
+Welcome! Have fun!
+movai@8245cb518b06:~/app$
+```
+
+---
 
-## Installing the Flow from sources
+## Runing from source code
```
Comment (AlexFernandes-MOVAI) [inline README.md:195]: running instead of `runing`

**Review by AlexFernandes-MOVAI — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (MithunKinarullathil) [reply README.md]: I thought we removed it, I will re-add.

File: README.md [hunk 4]
```diff
@@ -1,10 +1,31 @@
...
+</p>
 
-![teaser2](https://media.giphy.com/media/O8EalLOaPKzDv8bp22/giphy.gif)
+<p align="center">
+  <a href="https://github.com/MOV-AI/movai-flow/releases"><img alt="CircleCI" src="https://img.shields.io/github/release/MOV-AI/movai-flow.svg?label=current+release"></a>
+  <a href="https://github.com/MOV-AI/movai-flow/actions/workflows/DeployOnGitRelease.yml"><img alt="Deploy - To Nexus On Github Release" src="https://github.com/MOV-AI/movai-flow/actions/workflows/DeployOnGitRelease.yml/badge.svg"></a>
+  <a href="https://github.com/MOV-AI/movai-flow/actions/workflows/DeployOnMergeMain.yml"><img alt="Deploy - On branch main/release Push" src="https://github.com/MOV-AI/movai-flow/actions/workflows/DeployOnMergeMain.yml/badge.svg"></a>
+  <a href="https://github.com/MOV-AI/movai-flow/actions/workflows/TestOnPR.yml"><img alt="CI - On main/dev/release branches" src="https://github.com/MOV-AI/movai-flow/actions/workflows/TestOnPR.yml/badge.svg"></a>
+  <a href="https://twitter.com/MovAIRobots"><img alt="Twitter" src="https://img.shields.io/twitter/url/http/shields.io.svg?style=social"></a>
```
[the remark this answers is not in the document]
Comment (MithunKinarullathil) [reply README.md:10]: Need help.

[the remark this answers is not in the document]
Comment (thetaleofnarutouzumaki) [reply README.md:10]: Done.

**Review by AlexFernandes-MOVAI — APPROVED** [review]

**Pull request merged**